ctapipe/pipeline/algorithms/list_teldata.py

```diff
@@ -0,0 +1,17 @@
+from time import sleep
+import threading
+from ctapipe.core import Component
+
+
+class ListTelda(Component):
```

Comment:
rename this (doesn't follow naming conventions of using full words and capitalize each word, CapWordsStyle).  Maybe `ListTelsWithData`  or something like that.